meeting_summary/2021-11-27-确定性数据库.md

```diff
@@ -0,0 +1,52 @@
+# 确定型数据库概念
+
+确定型数据库是指在事务执行顺序确定的情况下，数据库的执行结果是唯一的。例如
+$$
+T_{1}<T_{2}<T_{3}
+$$
+那么事务执行的顺序就会按照，$T_{1},T_{2},T_{3}$的顺序进行执行。
```

Comment:
format error.
There is a solution https://gist.github.com/a-rodin/fef3f543412d6e1ec5b6cf55bf197d7b#gistcomment-3410708

![T_{1}<T_{2}<T_{3}](https://render.githubusercontent.com/render/math?math=%5Cdisplaystyle+T_%7B1%7D%3CT_%7B2%7D%3CT_%7B3%7D)